### ISSUE TITLE
Add Subscribed Playlist Videos to What's New Feed

### ISSUE THREAD

**Issue by kakolisgay** [TeamNewPipe/NewPipe#3525]
Could be useful for channels where users are only interested in a few series but not the other content.

**Comment by Stypox** [comment]
Duplicate of #2414

**Comment by kakolisgay** [comment]
@Stypox Are you sure? 2414 doesn't mention being able to subscribe to playlists

**Comment by Stypox** [comment]
Oh, I misread the title, sorry ;-)
Anyway, duplicate of #3210

**Comment by kakolisgay** [comment]
@Stypox Uh still nope, this request feature is allowing a user to subscribe to get playlist video's in their What's New subscription feed like you can do with channels. Not about playlist tabs

**Comment by Stypox** [comment]
Then it is roughly a duplicate of #1376, but I'll close that issue in favour of this since you provide a better description ;-)

**Comment by MD77MD** [comment]
this feature is a most... some times you don't want to subscribe to all the videos from a certain channel, your just interested in a playlist regarding a particular topic.

Instead of cluttering subscription feed by having all videos from that channel to show up. only new videos add to that particular playlist will show up in the subscription feed.

YouTube did add this feature for some time, but off course, as usual, they took it away from us. 

Anyway, this relies completely on (Add support for extracting multiple tabs from a channel #279 ). I just wanted to offer more usage information of this future and how can everyone benefit from implementing it.

**Comment by opusforlife2** [comment]
@MD77MD I don't think it relies on that at all. You can already see playlists in search. What is needed is just an option to _subscribe_ to a playlist the way you can do for channels. So an additional option in the long-press menu.

**Comment by MD77MD** [comment]
> @MD77MD I don't think it relies on that at all. You can already see playlists in search. What is needed is just an option to _subscribe_ to a playlist the way you can do for channels. So an additional option in the long-press menu.

your right... I didn't think of that. but it's kind hard to know if a channel has such playlists unless you visit there channel on YouTube app. usually users would just subscribe because it's easier then going through YouTube. that's why we really need playlists tabs.

**Comment by opusforlife2** [comment]
Of course we need channel tabs. It's a missing feature of many services regardless of anything else.

**Comment by nikhilCad** [comment]
Any progress?

**Comment by nikhilCad** [comment]
Copying from #1376 as I think its relevant

It could work similar to how you subscribe to channels, for example:
https://www.youtube.com/feeds/videos.xml?playlist_id=PLiuUQ9asub3Ta8mqP5LNiOhOygRzue8kN
If you put this in a RSS Reader, you will only see Hubble videos and not every video NASA uploads.
